app/_hub/kong-inc/opentelemetry/overview/_index.md

```diff
@@ -225,14 +238,57 @@ If no appropriate format is found, the plugin will fallback to the default forma
 {% endif_version %}
 
 
-### OTLP exporter
+#### OTLP exporter
 
 The OpenTelemetry plugin implements the [OTLP/HTTP](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/otlp.md#otlphttp) exporter, which uses Protobuf payloads encoded in binary format and is sent via HTTP/1.1.
 
 `connect_timeout`, `read_timeout`, and `write_timeout` are used to set the timeouts for the HTTP request.
 
 `batch_span_count` and `batch_flush_delay` are used to set the maximum number of spans and the delay between two consecutive batches.
 
+{% if_version gte:3.8.x %}
+### Logging
+
+This plugin supports [OpenTelemetry Logging](https://opentelemetry.io/docs/specs/otel/logs/), which can be configured as described in the [configuration reference](/hub/kong-inc/opentelemetry/configuration/#config-traces_endpoint) to export logs in OpenTelemetry format to an OTLP-compatible backend.
+
+#### Log scopes
+
+Two different kinds of logs are exported: **Request** and **Non Request** scoped.
+The former are logs that are directly associated with requests, they are produced during the request lifecycle. Example: logs generated during a plugin's Access phase.
+The latter are logs that are not directly associated with a request, they are produced outside the request lifecycle. Examples logs generated asynchronously or during a worker's startup.
```

Comment:
```suggestion
  * Request logs are directly associated with requests. They are produced during the request lifecycle. For example, this could be logs generated during a plugin's Access phase.
  * Non-request logs are not directly associated with a request. They are produced outside the request lifecycle. For examples, this could be logs generated asynchronously or during a worker's startup.
```